app/controllers/mixins/checked_id_mixin.rb

```diff
@@ -123,9 +123,7 @@ def find_record_with_rbac(klass, id, options = {})
     #   either sets flash or raises exception
     #
     def find_records_with_rbac(klass, ids, options = {})
-      filtered = Rbac.filtered(klass.where(:id => ids),
-                               :user        => current_user,
```

Comment:
current_user is not necessary, as it loaded in https://github.com/ManageIQ/manageiq/blob/master/lib/rbac/filterer.rb#L496 by default